### ISSUE TITLE
word-autocorrelation code might be too complicated for 20-30 minute breakout room

### ISSUE THREAD

**Issue by wikfeldt** [ENCCS/HPDA-Python#59]
consider having simpler examples, and have autocorrelation as optional exercises or as "project work" on last day

**Comment by wikfeldt** [comment]
Word-acf is still a nice type of example because it's somewhat realistic as a scientific problem and maybe corresponds to how people often write python, so maybe best approach is to simplify by removing all the setup/boilerplate and only focus on the two core functions. I.e. use pre-cleaned data etc

**Comment by qianglise** [comment]
i agree, we should extract two separate py files, one for word preprocessing and another for word-acf and use them as examples.

**Comment by wikfeldt** [comment]
similarly to what's being done to the [Julia lesson](https://enccs.github.io/Julia-for-HPC/), the word-acf example can move into a separate exercises episode where all techniques learned can be applied to the same problem
